.markdownlint-cli2.yaml

```
@@ -13,7 +13,8 @@ globs:
   - "**/*.md"
 ignores:
   - ".tox/**"
-  - "venv/**"
   - ".venv/**"
   - "build/**"
+  - "schema/**"
```

```suggestion
  - "src/instructlab/schema/**"
```

This needs to use the folder of the schema submodule in this repo.

---

Doh, copy/paste! :-D